blazor/gantt-chart/data-binding.md

```diff
@@ -559,6 +559,292 @@ In the below example, `TaskData` implements `INotifyPropertyChanged` and it rais
 ```
 ![Property changed in Blazor Gantt Chart](images/blazor-gantt-chart-observable-property-changed.PNG)
 
+### Load Child on Demand
+
+To render child records on demand, assign service data as an instance of **SfDataManager** to the [DataSource](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_DataSource) property. To interact with remote data source,  provide the endpoint **url** and define the [GanttTaskFields.HasChildMapping](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.GanttTaskFields.html#Syncfusion_Blazor_Gantt_GanttTaskFields_HasChildMapping) property of Gantt Chart.
+
+The `GanttTaskFields.HasChildMapping` property maps the field name in data source, that denotes whether current record holds any child records. This is useful internally to show expand icon while binding child data on demand.
+
+When [LoadChildOnDemand](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_LoadChildOnDemand) is disabled, all the root nodes are rendered in collapsed state at initial load. On expanding the root node, the child nodes will be loaded from the remote server.
+
+When a root node is expanded, its child nodes are rendered and maintained in a collection locally, such that on consecutive expand/collapse actions on root node, the child nodes are loaded locally instead from the remote server.
```

Comment:
mention only current view port child nodes are fetched from server and rendered in DOM

Reply:
Updated as suggested.